### PR TITLE
proxmox_kvm: trivial patch for github issue #1875

### PR DESCRIPTION
**Pull request by Aversiste**
##### SUMMARY

Fixes issue #1875: parameter `vmid` passed twice to `exit_json()`.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

proxmox_kvm

### CODE REVIEW

**Comment by felixfontein** [comment]
@Aversiste can you add a changelog fragment?

File: changelogs/fragments/1895-proxmox-kvm-fix-issue-1875.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - proxmox_kvm - fix parameter ``vmid`` passed twice to ``exit_json`` while creating a virtual machine without cloning, https://github.com/ansible-collections/community.general/pull/1895
```
Comment (felixfontein) [inline changelogs/fragments/1895-proxmox-kvm-fix-issue-1875.yml:3]:
```suggestion
  - proxmox_kvm - fix parameter ``vmid`` passed twice to ``exit_json`` while creating a virtual machine without cloning (https://github.com/ansible-collections/community.general/issues/1875, https://github.com/ansible-collections/community.general/pull/1895).
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
It's probably a good idea to eventually get rid of that global variable, but for now this is the best way to fix the situation.

**Comment by felixfontein** [comment]
@Aversiste thanks for fixing this!

**Comment by Aversiste** [comment]
> It's probably a good idea to eventually get rid of that global variable, but for now this is the best way to fix the situation.

Oh yes.

**Comment by bubthegreat** [comment]
This needs to also be changed https://github.com/ansible-collections/community.general/blob/5c60954980e37a57f2bf4c337e2a1497513026ec/plugins/modules/cloud/misc/proxmox_kvm.py#L1286 and https://github.com/ansible-collections/community.general/blob/5c60954980e37a57f2bf4c337e2a1497513026ec/plugins/modules/cloud/misc/proxmox_kvm.py#L1288

**Comment by felixfontein** [comment]
@bubthegreat why do these places need to be changed? They only return `vmid` once, not twice.

**Comment by bubthegreat** [comment]
When I was cloning a cloud init template that it was erroring with the stack trace, so I modified both the clone statement and the update statement since `module.exit_json` appeared to be handled the same way in all three cases.  That fixed the issue, where only applying this patch locally did not - but I might have screwed something up.  If I can reproduce it locally when I have time to go back over the home infrastructure I will - I don't have my server stack up and running until I finish moving to a new house in a week or two.

**Comment by felixfontein** [comment]
@bubthegreat the problem was the `**result`, which was only present in one `exit_json` call (which this PR fixed).

**Pull request merged**